CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+### Removed 
+* Progressblock verwijderen (#802)

```

Comment:
Strikt genomen moet hier dan nog een `**BREAKING**: ` voor toch? En, zouden we in deprecations en removeds niet ook nog naar de PR verwijzen? Dus zoiets als 
`**Removal notice, see PR #814**`

@tfrijsewijk ?

Reply:
Ja, embrace semver